integration/integration_test.go

```diff
@@ -222,7 +222,7 @@ func testAuditOn(t *testing.T, suite *integrationTestSuite) {
 	tr := utils.NewTracer(utils.ThisFunction()).Start()
 	defer tr.Stop()
 
-	var tests = []struct {
+	tests := []struct {
```

Comment:
These changes were applied automatically via gofumpt - happy to revert if anyone prefers the longer var declaration.